Database/Corrections/sodQuestFixes.lua

```diff
@@ -2358,6 +2358,66 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -424992,
             [questKeys.zoneOrSort] = sortKeys.ROGUE,
         },
+        [90139] = {
+            [questKeys.name] = "Quick Strike",
+            [questKeys.startedBy] = {{1161,1162,1163,1164}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 14,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.WARRIOR,
+            [questKeys.objectivesText] = {"Kill Troggs for the Skull-Shaped Geode. Then get hit by a Skullthumper's stun and use the geode to loot the rune."},
+            [questKeys.requiredSpell] = -425443,
+            [questKeys.zoneOrSort] = sortKeys.WARRIOR,
+        },
+        [90140] = {
+            [questKeys.name] = "Quick Strike",
+            [questKeys.startedBy] = {{504,95,449,590,589,121,285,46,732,735},{408799}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 20,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.WARRIOR,
+            [questKeys.objectivesText] = {"Kill Defias for Gillsbane, equip it and then kill Murlocs to receive Soul of the Sea. Take it to the Idol of the Deep to summon Horror of the Deep and defeat it."},
+            [questKeys.requiredSpell] = -425443,
+            [questKeys.zoneOrSort] = sortKeys.WARRIOR,
+        },
+        [90141] = {
+            [questKeys.name] = "Quick Strike",
+            [questKeys.startedBy] = {nil,{408802}},
+            [questKeys.finishedBy] = {210482},
```

Comment:
For fake quests the `finishedBy` field is useless at this point. Questie only reads it on regular quests after all objectives are completed. So to show a location for this quest add it to `startedBy`.

```suggestion
            [questKeys.startedBy] = {{210482},{408802}},
            [questKeys.finishedBy] = nil,
```

Also I am not sure if https://www.wowhead.com/classic/object=408802/gnarled-harpoon has more than one spawn, even though wowhead lists more than one on that page. Might be worth to verify and cleanup the spawns, to reduce the amount of rune icons on the map.

Reply:
Fixed in :https://github.com/Questie/Questie/pull/5435/commits/bcc1833185b042373344bdbeac2d47c3a53f6b27

I can confirm it has multiple spawn spawns like shown on WoWHead. I've done it a couple of times myself, and just checked with guildies that this has happened to them too.